assets/js/buttons-examples.js

```diff
@@ -154,6 +154,11 @@ $(document).ready(function () {
           label: "Transparent On Hover",
           classes: "transparent-btn",
         },
+        {
```

Comment:
Please place the button in the correct alphabetical order

Reply:
Sure done.
Thanks